src/build/validator.md

```diff
@@ -422,49 +471,43 @@ An extension of the sentry node architecture optionally sees a Validator operato
 
 Beyond the set up of a server, a node, an authenticated way of joining the Cudos blockchain using our in-built public key infrastructure, in coordination with Ledger HSM or YubiHSM for those Validators choosing to implement them, the use of full nodes when interacting the network is highly recommended. We plan to implement the ability for Cudos Validator Nodes to store a history of previously signed blocks in order to more seamlessly prevent double-signing by adverse or deficient nodes in the Cudos Network. This feature is currently absent in earlier-generation Tendermint blockchains. The final element keeping Cudos Network Validating nodes safe is the Tendermint Core Byzantine Fault Tolerant Proof of Stake consensus algorithm.
 
-## Hardware Requirements
+## Hardware requirements
+
+When having a network of 3 validators:
 
-The following are the minimum hardware requirements Validators will need to run to successfully join and operate on the Cudos Network for the Testnet and Mainnet (Version 1) stage:
+- AMD Rome 2 vCPUs, 4 GB memory
+- Cudos-root-node-disk - 320 SSD
+- CPU Utilization: ~5%
+- Memory Utilization: ~15%
+- Network Utilization: ~5KiB/s
+- Disk: ~ 3 GB for 31 days in current environment
 
-#### Cudos mainnet ("Ingenii") Validator node
+You must have the minimum hardware requirements for each node that you are running on the Testnet network as explained below (extrapolating network to 125 Validators).
```

Comment:
we have a cap of 100 validators but I don't know why we would have to reference that number here